api_tests/nodes/filters/test_filters.py

```diff
@@ -94,3 +94,103 @@ def test_contributor_filter(self):
         res = self.app.get('{}{}'.format(self.contributors_url, self.user_two._id), auth=self.user.auth)
         actual = [node['id'] for node in res.json['data']]
         assert_equal(expected, actual)
+
+
+class NodesListDateFilteringMixin(object):
```

Comment:
Probably this should be at the top of the file rather than the bottom based on today's discussion. Mixins at the top make sense. Perhaps we'll add that to the cosdev docs one day.